controllers/managedocs_controller.go

```diff
@@ -925,26 +1008,28 @@ func (r *ManagedOCSReconciler) reconcilePrometheusService() error { //nolint:dea
 
 		r.prometheusService.Spec.Ports = []corev1.ServicePort{
 			{
-				Name:       "odf-prometheus-endpoint",
+				Name:       "https",
 				Protocol:   corev1.ProtocolTCP,
 				Port:       9339,
-				TargetPort: intstr.FromString("web"),
+				TargetPort: intstr.FromString("https"),
 			},
 		}
 		r.prometheusService.Spec.Selector = map[string]string{
-			monLabelKey: monLabelValue,
+			prometheusServiceSelectorLabelKey: prometheusServiceSelectorLabelValue,
 		}
 		utils.AddAnnotation(r.prometheusService,
 			servingCertSecretBetaAnnotationKey, prometheusServiceSecretName)
 		utils.AddAnnotation(r.prometheusService,
 			servingCertSecretAlphaAnnotationKey, prometheusServiceSecretName)
```

Comment:
These are known annotations defined outside of our product, so I see no point in hiding them in a constant as they are not going to change throughout the lifetime of the operator.

Also, long lines should be split in accordance with this repo coding style

 ```suggestion
		utils.AddAnnotation(
			r.prometheusService,
			"service.beta.openshift.io/serving-cert-secret-name", 
			prometheusServiceSecretName,
		)
		utils.AddAnnotation(
			r.prometheusService,
			"service.alpha.openshift.io/serving-cert-secret-name", 
			prometheusServiceSecretName,
		)
```